projects/novo-elements/src/elements/data-table/cell-headers/data-table-header-cell.component.ts

```diff
@@ -326,12 +368,79 @@ export class NovoDataTableCellHeader<T> implements IDataTableSortFilter, OnInit,
   public cancel(): void {
     this.multiSelectedOptions = this.filter ? [...this.filter] : [];
     this.dropdown.closePanel();
+    this.clearOptionFilter();
   }
 
   public filterMultiSelect(): void {
-    let actualFilter = this.multiSelectedOptions.length > 0 ? [...this.multiSelectedOptions] : undefined;
-    this.filterData(actualFilter);
-    this.dropdown.closePanel();
+    if ((this.multiSelectedOptions.length === 0 && !this.filter) || this.multiSelectedOptions === this.filter) {
```

Comment:
If `this.multiSelectedOptions` is an array, the equality comparison against `this.filter` will always fail, and you need to use a helper function to sort & compare the elements of each array. Gonna DM you some novo code that handles this, if this turns out to be a necessary check.

Reply:
Thanks for calling this out, it actually isn't necessary now that I think about it